test/basic.html

```diff
@@ -236,6 +247,59 @@
 
   });
 
+  suite('resetting', function () {
+    test('form has the right default values', function() {
+      var form = fixture('FormForResetting');
+
+      assert.equal(form._customElements.length, 1);
+      assert.equal(form.elements.length, 3);
+
+      assert.equal(form._customElementsInitialValues.length, 1);
+      assert.equal(form._customElementsInitialValues[0], 'zag');
```

Comment:
Since the next test actually sets the resetting of values, this test isn't really needed